geopatterns_demo/api.py

```diff
@@ -46,10 +46,22 @@ def generate(
     )
 
 
-@api.get('/methods')
+@api.get(
+    '/methods',
+    response_model=typing.List[models.MethodDescription],
+)
 def methods():
     """Generate and return list of the methods geopatterns supports."""
-    return describe_available_methods()
+    return methods_list.describe_available_methods()
+
+
+@api.get(
+    '/random-phrase',
+    response_model=models.PhraseDescription,
+)
+def phrase():
+    """Generate and return return random phrase."""
+    return {'text': random_phrase.make_random_text()}
```

Comment:
Since there is a `response_model` defined, it is easier to just return an instance of this model, no need to return a raw `dict`. Try this:

```suggestion
    return models.PhraseDescription(text=random_phrase.make_random_text())
```